extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-cta-button.css

```diff
@@ -4,14 +4,20 @@
  in story-ad-button-text-fitter.js */
 .i-amphtml-story-ad-link {
   background-color: #ffffff !important;
-  border-radius: 20px !important;
+  border-radius: 4em !important;
   box-sizing: border-box !important;
   bottom: 32px !important;
   box-shadow: 0px 2px 12px rgba(0, 0, 0, 0.16) !important;
   color: #4285f4 !important;
   font-family: 'Roboto', sans-serif !important;
   font-weight: bold !important;
-  height: 36px !important;
+  /* dymnamic sizing for text */
+  font-size: clamp(min(15px), 2vmin, max(3vmin)) !important;
```

Comment:
sorry its been a while since I have looked at this code, can you confirm this doesn't mess with the intended behavior of `story-ad-button-text-fitter.js`?